### PR TITLE
Fix: Readme and Contributing Guide

### PR DESCRIPTION
**Pull request by jessica-tw**
Signed-off-by: jessica-tw <jessica.olinda@zup.com.br>

**- What I did**
- Fixed Readme file.
- Added new Contributing Guide.

**- How to verify it**

**- Description for the changelog**  Fix: Readme and Contributing Guide

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,91 +1,175 @@
...
-This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 1.4,
-available at [http://contributor-covenant.org/version/1/4][version]
+**Example:** **[start][bugfix]: Fix bug when Horusec haven't read the new flag of authorization**
 
-[homepage]: http://contributor-covenant.org
-[version]: http://contributor-covenant.org/version/1/4/
+ 
+2. Answer the questions about what you did, how to verify it and a short description for the changelog, see an example below:
+
+<p align="center" margin="20 0"><img src="assets/pr-template.PNG" alt="architecture" width="100%" style="max-width:100%;"/></p>
```
Comment (matheusalcantarazup) [inline CONTRIBUTING.md:89]: This image was not added on commit.

**Review by matheusalcantarazup — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,91 +1,176 @@
...
+- 2 code owners approve it.
+- Pass all GitHub actions checking process (lint, test, coverage, license, build, e2e, security, dco).
+
+### **After your pull request's approval**
+- If it is a bug fix, the team will perform the changes and there will be a new release.
+- If it is a feature, it will be in the next release. 
+
+## **Community**
+
+- Do you have any question about Horusec? Send to out [**mailing list**](horusec@zup.com.br).
```
Comment (nathanmartinszup) [inline CONTRIBUTING.md:171]: Link returning not found

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,91 +1,176 @@
...
+```bash
+make coverage
+
+go tool cover -html=coverage.out # Open coverage status in your browser
+``` 
+
+#### **Repositories**
+Horusec has other repositories and you can check the README for each one of them: 
+
+- [**Charts**](https://github.com/ZupIT/charlescd/tree/main/circle-matcher)
```
[the remark this answers is not in the document]
Comment (wiliansilvazup) [reply CONTRIBUTING.md:125]: ops! @jessica-tw change to use horusec links :)

File: CONTRIBUTING.md (hunk 2 above)
Comment (jessica-tw) [reply CONTRIBUTING.md:171]: Do you guys use this email?

File: CONTRIBUTING.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (jessica-tw) [reply CONTRIBUTING.md:125]: Changed! ;)

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,91 +1,176 @@
...
+- [**Devkit**](https://github.com/ZupIT/horusec-devkit)
+- [**Engine**](https://github.com/ZupIT/horusec-engine)
+- [**Jenkins**](https://github.com/ZupIT/horusec-jenkins-sharedlib)
+- [**Operator**](https://github.com/ZupIT/horusec-operator)
+- [**Platform**](https://github.com/ZupIT/horusec-platform)
+- [**VSCode plugin**](https://github.com/ZupIT/horusec-vscode-plugin)
+- [**Kotlin**](https://github.com/ZupIT/horusec-tree-sitter-kotlin)
```
[the remark this answers is not in the document]
Comment (jessica-tw) [reply CONTRIBUTING.md:132]: Which one do we need to remove?

File: CONTRIBUTING.md (hunk 2 above)
Comment (nathanmartinszup) [reply CONTRIBUTING.md]: The email is correct, the problem was only in the link but it has already been fixed.

File: CONTRIBUTING.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (nathanmartinszup) [reply CONTRIBUTING.md:133]: Here's the updated repositories list:

https://github.com/ZupIT/horusec
https://github.com/ZupIT/horusec-devkit
https://github.com/ZupIT/horusec-platform
https://github.com/ZupIT/horusec-engine
https://github.com/ZupIT/horusec-vscode-plugin
https://github.com/ZupIT/horusec-operator
https://github.com/ZupIT/horusec-docs
https://github.com/ZupIT/horusec-c4model
https://github.com/ZupIT/horusec-examples-vulnerabilities

[the remark this answers is not in the document]
Comment (nathanmartinszup) [reply CONTRIBUTING.md:133]: Different than that should be removed

[the remark this answers is not in the document]
Comment (jessica-tw) [reply CONTRIBUTING.md:133]: Nice, I will fix it in every repo. Thanks ;)

**Pull request merged**